phpseclib/Net/SSH2.php

```diff
@@ -204,7 +204,7 @@ class SSH2
     /**
      * The Socket Object
      *
-     * @var object
+     * @var resource|closed-resource|null
```

Comment:
Didn't know closed-resource was a distinctly different thing from resource lol

Reply:
closed-resource is a psalm specific type used to catch bugs:
```php
// is a resource
$this->fsock = \fsockopen(...);

// now it is a closed-resource
\fclose($this->fsock);

// Psalm would throw an error here now
fputs($this->fsock, ...);
```